plugins/generator-1.18.2/forge-1.18.2/templates/livingentity/livingentity_renderer.java.ftl

```diff
@@ -131,6 +131,21 @@ public class ${name}Renderer extends <#if humanoid>Humanoid</#if>MobRenderer<${n
 	    }
 	</#if>
 
+	<#if data.mobScale??>
+	@Override protected void scale(${name}Entity _ent, PoseStack poseStack, float p_116462_) {
+		<#if hasProcedure(data.mobScale)>
+			Entity entity = _ent;

```

Comment:
No need to complicate things (© KlemenDEV), you can justrename `_ent` parameter to `entity` and then this parameter will be used by the procedure.

Reply:
Would that really work? Renderer provides {Name}Entity, while we just need a normal Entity for the procedure. Wouldn't it cause issues if the provided entity is not already of type Entity? Additionally, every other method in renderer creates this sub variable @Defeatomizer

Reply:
`` is a (sub)type of `Entity` so I don't see how this can not be working.

Reply:
Is there any reason why the rest of the methods do this then? @Defeatomizer

Reply:
That was a mistake on our side, and it does not mean we or you should repeat that mistake (correct me if I'm wrong, maintainers).

Reply:
yeah just renaming the argument to entity and deleting the local variable should work better